src/components/date-picker/flatpickr-adapter/flatpickr-adapter.tsx

```diff
@@ -144,7 +144,8 @@ export class DatePickerCalendar {
         const ONE_SECOND = 1000;
         const TEN_PIXELS = 10;
         setTimeout(() => {
-            const { width } = this.container.getBoundingClientRect();
+            const flatpickrElement = this.container.querySelectorAll('div.flatpickr-calendar')[0];
```

Comment:
```suggestion
            const flatpickrElement = this.container.querySelector('div.flatpickr-calendar');
```

This will both be faster, and not throw an error if nothing is found (although, an error will still be thrown on the next line if that happens…)